src/components/Home/style.js

```diff
@@ -21,7 +21,7 @@ export const useStyles = makeStyles({
     borderRadius: '0 0 0 10%',
     background: '#B2E3FF',
     zIndex: '-2',
-    left: '1033px',
-    bottom: '300px',
+    left: '1105px',
+    bottom: '328px',
```

Comment:
bad idea , use some relative units. Fixed units are quite ok if  small values are used, for example for some paddings/margins